pkg/cluster/describe.go

```diff
@@ -130,6 +130,11 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 		privateLinkEnabled = cluster.AWS().PrivateLink()
 	}
 
+	stsEnabled := false
+	if cluster.AWS().STS() != nil {
```

Comment:
Do we need nil validation for 

 if cluster.AWS()!=nil  &&  cluster.AWS().STS() != nil

Reply:
Good point, let me put it in the same block as the privateLink check above

Reply:
No, this should be fine, as the types are initialized. However, this also means that checking for `STS != nil` is not sufficient, you'll need something like:
```go
if cluster.AWS().STS().RoleARN() != "" {
  stsEnabled = true
}
```